components/asic/bm1366.c

```diff
@@ -141,72 +126,26 @@ void BM1366_set_version_mask(uint32_t version_mask)
 
 void BM1366_send_hash_frequency(float target_freq)
 {
-    // default 200Mhz if it fails
-    unsigned char freqbuf[9] = {0x00, 0x08, 0x40, 0xA0, 0x02, 0x41}; // freqbuf - pll0_parameter
-    float newf = 200.0;
-
-    uint8_t fb_divider = 0;
-    uint8_t post_divider1 = 0, post_divider2 = 0;
-    uint8_t ref_divider = 0;
-    float min_difference = 10;
-
-    // refdiver is 2 or 1
-    // postdivider 2 is 1 to 7
-    // postdivider 1 is 1 to 7 and less than postdivider 2
```

Comment:
This was changed from postdiv2 < postdiv1 to allowing them to be the same. I think this was an unintended change.